packages/devtools_app/lib/src/screens/performance/panes/queued_microtasks/queued_microtasks_view.dart

```diff
@@ -0,0 +1,278 @@
+
+  @override
+  Widget build(BuildContext context) {
+    return Center(
+      child: RichText(
+        text: TextSpan(
+          style: Theme.of(context).regularTextStyle,
+          children: [
+            const TextSpan(text: 'Click the refresh button '),
+            WidgetSpan(child: Icon(Icons.refresh, size: defaultIconSize)),
+            const TextSpan(
+              text:
+                  " to take a new snapshot of the selected isolate's "
+                  'microtask queue.',
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+// In the response returned by the VM Service, microtasks are sorted in
+// ascending order of when they will be dequeued, i.e. the microtask that will
+// run earliest is at index 0 of the returned list. We use those indices of the
+// returned list to sort the entries of the microtask selector, so that they
+// they also appear in ascending order of when they will be dequeued.
+typedef IndexedMicrotask = (int, Microtask);
+
+class _MicrotaskIdColumn extends ColumnData<IndexedMicrotask> {
+  _MicrotaskIdColumn()
+    : super.wide('Microtask ID', alignment: ColumnAlignment.center);
+
+  @override
+  int getValue(IndexedMicrotask indexedMicrotask) => indexedMicrotask.$1;
+
+  @override
+  String getDisplayValue(IndexedMicrotask indexedMicrotask) =>
+      indexedMicrotask.$2.id!.toString();
+}
+
+class QueuedMicrotaskSelector extends StatelessWidget {
+  const QueuedMicrotaskSelector({
+    super.key,
+    required List<IndexedMicrotask> indexedMicrotasks,
+    required void Function(Microtask?) setSelectedMicrotask,
+  }) : _indexedMicrotasks = indexedMicrotasks,
+       _setSelectedMicrotask = setSelectedMicrotask;
+
+  static final _idColumn = _MicrotaskIdColumn();
+  final List<IndexedMicrotask> _indexedMicrotasks;
+  final void Function(Microtask?) _setSelectedMicrotask;
+
+  @override
+  Widget build(BuildContext context) => Column(
+    crossAxisAlignment: CrossAxisAlignment.start,
+    children: [
+      Expanded(
+        child: FlatTable<IndexedMicrotask>(
+          keyFactory: (IndexedMicrotask microtask) =>
+              ValueKey<int>(microtask.$1),
+          data: _indexedMicrotasks,
+          dataKey: 'queued-microtask-selector',
+          columns: [_idColumn],
+          defaultSortColumn: _idColumn,
+          defaultSortDirection: SortDirection.ascending,
+          onItemSelected: (indexedMicrotask) =>
+              _setSelectedMicrotask(indexedMicrotask?.$2),
+        ),
+      ),
+    ],
+  );
+}
+
+class StackTraceView extends StatelessWidget {
+  const StackTraceView({super.key, required selectedMicrotask})
+    : _selectedMicrotask = selectedMicrotask;
+
+  final Microtask? _selectedMicrotask;
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context);
+
+    return Column(
+      children: [
+        SizedBox.fromSize(
+          size: Size.fromHeight(defaultHeaderHeight),
+          child: Container(
```

Comment:
nit: remove the parent SizedBox and just set `height: defaultHeaderHeight` on Container.